__tests__/api-categories.test.js

```diff
@@ -15,31 +15,30 @@ afterAll(() => {
 });
 
 describe("1. GET /api/categories", () => {
-      test("200: GET /api/categories returns an array", () => {
-				return request(app)
-					.get("/api/categories")
-					.expect(200)
-					.then((res) => {
-                        const { body } = res;
-                        console.log(body.categories, "body")
-						expect(Array.isArray(body.categories)).toBe(true);
+    test("200: GET /api/categories returns an array of categories", () => {
+        return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then((res) => {
+                const { body } = res;
+                console.log(body.categories, "body")
+                expect(Array.isArray(body.categories)).toBe(true);
+            }).then(() => {
+                return request(app)
```

Comment:
This is nicer, and works fine, but I think you could work them both into one return request(app) section. I shall approve for now but if you decide to refactor later, please submit another PR.